packages/node_modules/@webex/plugin-meetings/src/metrics/index.js

```diff
@@ -1,46 +1,534 @@
+    this.postEvent(options);
+  }
+
+  mediaReconnected(options) {
+    options.event = eventType.CLIENT_MEDIA_RECONNECTING;
+    this.postEvent(options);
+  }
+
+  mediaRecovered(options) {
+    options.event = eventType.CLIENT_MEDIA_RECOVERED;
+    this.postEvent(options);
+  }
+
+  mediaReconnectionFailed(options) {
+    options.event = eventType.CALL_RECONNECTION_FAILED;
+    this.postEvent(options);
+  }
+
+  pinPrompted(options) {
+    options.event = eventType.CLIENT_PIN_PROMPT;
+    this.postEvent(options);
+  }
+
+  pinCollected(options) {
+    options.event = eventType.CLIENT_PIN_COLLECTED;
+    this.postEvent(options);
+  }
+
+  enteredLobby(options) {
+    options.event = eventType.CLIENT_LOBBY_ENTERED;
+    this.postEvent(options);
+  }
+
+  exitedLobby(options) {
+    options.event = eventType.CLIENT_LOBBY_EXITED;
+    this.postEvent(options);
+  }
+
+  shareInitiated(options) {
+    options.event = eventType.CLIENT_SHARE_INITIATED;
+    this.postEvent(options);
+  }
+
+  shareStopped(options) {
+    options.event = eventType.CLIENT_SHARE_STOPPED;
+    this.postEvent(options);
+  }
+
+  shareFloorGranted(options) {
+    options.event = eventType.CLIENT_SHARE_FLOOR_GRANTED_LOCAL;
+    this.postEvent(options);
+  }
+
+  shareLayout(options) {
+    options.event = eventType.CLIENT_SHARE_LAYOUT_DISPLAYED;
+    this.postEvent(options);
+  }
+
+  muted(options) {
+    options.event = eventType.CLIENT_MUTED;
+    this.postEvent(options);
+  }
+
+  unmuted(options) {
+    options.event = eventType.CLIENT_UNMUTED;
+    this.postEvent(options);
+  }
+
+  leaveMeeting(options) {
+    options.event = eventType.CLIENT_CALL_LEAVE;
+    this.postEvent(options);
+  }
+
+  remoteEnded(options) {
+    options.event = eventType.CLIENT_CALL_REMOTE_ENDED;
+    this.postEvent(options);
+  }
+
+  remoteStarted(options) {
+    options.event = eventType.CLIENT_CALL_REMOTE_STARTED;
+    this.postEvent(options);
+  }
+
+  // ???? check if its correct
+  receivedIncoming(options) {
+    options.event = eventType.CLIENT_CALL_REMOTE_STARTED;
+    this.postEvent(options);
+  }
+
+  sendMediaRequest(options) {
+    options.event = eventType.CLIENT_LOCUS_MEDIA_REQUEST;
+    this.postEvent(options);
+  }
+
+  gotMediaResponse(options) {
+    options.event = eventType.CLIENT_LOCUS_MEDIA_RESPONSE;
+    this.postEvent(options);
+  }
+
+  startPstn(options) {
+    options.event = eventType.CLIENT_PSTN_AUDIO_ATTEMPT_START;
+    this.postEvent(options);
+  }
+
+  finishPstn(options) {
+    options.event = eventType.CLIENT_PSTN_AUDIO_ATTEMPT_FINISH;
+    this.postEvent(options);
+  }
+
+  skipPstn(options) {
+    options.event = eventType.CLIENT_PSTN_AUDIO_ATTEMPT_SKIP;
+    this.postEvent(options);
+  }
+
+  generateMediaEngineError() {
+
+  }
+
+  initPayload(eventType, identifiers, options) {
+    // TODO: add envirnoment to the call analyzer metrics
+
+    // const buildType = config.env === 'production' || config.env === 'test' ?
```

Comment:
Remove? Is this related to what you asked @taymoork2 about earlier?

Reply:
@mattnorris  @taymoork2  i was not sure if we can directly use the `process.env.NODE_ENV` directly there should be a global config file to determine it